shell/components/nav/Header.vue

```diff
@@ -889,6 +897,7 @@ export default {
 
     .rd-header-right {
       display: flex;
+      align-items: center;
```

Comment:
It looks like this mutates the layout of the namespace selector. We may need to revert this change and reintroduce the custom padding..